core/wasm/build.rs

```diff
@@ -0,0 +1,5 @@
+//! Build script
+
+fn main() {
+    println!("cargo:rustc-link-arg=--export-table");
```

Comment:
I've found out, that it's hard to google, what does this line do. Can you leave some comments about it?

For me it seems like you are asking linker to export table of exported functions, but I thought it is exported always, isn't it?